pkg/server/server.go

```diff
@@ -129,6 +129,25 @@ func NewServer(ctx context.Context, cfg *config.Config, logger *zap.Logger, name
 			return
 		}
 
+		nscs, nerr := srv.ConfigManager.ListAllNamespace(ctx)
```

Comment:
Why add a default namespace before `ImportNamespaceFromDir`? What if the user specifies a directory?

Reply:
I'll remove `--namespaces` from server, but add it to the cli later.

Reply:
Why? There is no way to specify an initial namespace. Maybe the user doesn't want a default one.

Reply:
> Why? There is no way to specify an initial namespace. 

This is "good default preset". And if you need something different, use ctl to config it. I don't want to be implicit that `--namespaces` will override the default fallback.

> Maybe the user doesn't want a default one.

We can delete the default namespace using `weirctl` whenever we want. If we want to delete it permanently, then we must only init the fallback only once, i.e. detect the workdir.

Reply:
It is now changed to:

1. give `default` fallback on the first run by detecting `cfg.Workdir`
2. `./weirctl namespace import conf/namespace` if yout need to init something different
3. `./weirctl namespace del default` if you don't want the fallback
4. `""` is renamed to `"default"`. And empty namespace name is forbidden

Reply:
Is `weirctl` available in the cloud?

Reply:
> Is `weirctl` available in the cloud?

Honestly, I don't know. I haven't used DBAAS. But I think `pd ctl` is avaiable, so it should be avaiable.